components/thumbnail/stories/thumbnail.stories.js

```diff
@@ -50,12 +60,6 @@ export default {
 			table: {
 				type: { summary: "string" },
 				category: "Content",
-			/* The `isSelected` property in the tree view component is used to indicate whether a specific item
-			in the tree view is currently selected or not. When `isSelected` is set to `true` for an item, it
-			visually highlights that item as selected, providing a visual cue to the user that it is the
-			currently active or focused item in the tree view. This property helps improve user experience by
-			making it easier for users to identify which item they are interacting with within the tree view
-			structure. */
```

Comment:
I don't know why this was here, it seemed like it was put here by mistake so I removed it.